<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>689</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>K-A-A</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourse.qrc">
    <normaloff>:/boa/resourse/udaf.png</normaloff>:/boa/resourse/udaf.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="accessibleName">
   <string>KAA</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/menu/resourse/bg.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="menu">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>105</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>105</width>
      <height>80</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/menu/resourse/b_menu.png</normaloff>:/menu/resourse/b_menu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>105</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="heart_1">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>110</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/menu/resourse/hrt.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="heart_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>110</y>
      <width>99</width>
      <height>103</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/menu/resourse/hrt1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="heart_3">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>110</y>
      <width>89</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/menu/resourse/hrt3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ans_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>73</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/menu/resourse/numero1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ans_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>101</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/menu/resourse/numero2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ans_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>91</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/menu/resourse/numero3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ans_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>73</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/menu/resourse/numero4.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>159</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>button_1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>159</width>
      <height>221</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>159</width>
      <height>221</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>148</red>
          <green>148</green>
          <blue>148</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>button_2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>410</y>
      <width>159</width>
      <height>221</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>148</red>
          <green>148</green>
          <blue>148</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>button_3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butt_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>410</y>
      <width>159</width>
      <height>221</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>148</red>
          <green>148</green>
          <blue>148</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>button_4</string>
    </property>
   </widget>
   <widget class="QLabel" name="tail">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>579</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/boa/resourse/udaf1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="hero">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Hero</string>
    </property>
   </widget>
   <widget class="QLabel" name="quote">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Quote</string>
    </property>
   </widget>
   <widget class="QLabel" name="quizz">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>501</width>
      <height>297</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="level">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>0</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="head">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>197</width>
      <height>101</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>232</red>
          <green>232</green>
          <blue>232</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>232</red>
          <green>232</green>
          <blue>232</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>232</red>
          <green>232</green>
          <blue>232</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/boa/resourse/udaf.png</normaloff>:/boa/resourse/udaf.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>197</width>
      <height>101</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="ans">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>501</width>
      <height>297</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>232</red>
          <green>232</green>
          <blue>232</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>232</red>
          <green>232</green>
          <blue>232</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>232</red>
          <green>232</green>
          <blue>232</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="med">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>281</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/boa/med.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>551</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/menu/resourse/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="go_away">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>375</width>
      <height>92</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/menu/resourse/go_away.png</normaloff>:/menu/resourse/go_away.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>375</width>
      <height>92</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="new_game_but">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>493</width>
      <height>83</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/menu/resourse/new_game.png</normaloff>:/menu/resourse/new_game.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>493</width>
      <height>83</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="lvl_0">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>148</red>
          <green>148</green>
          <blue>148</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/menu/0.png</normaloff>:/menu/0.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="lvl_1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>148</red>
          <green>148</green>
          <blue>148</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/menu/1.png</normaloff>:/menu/1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="lvl_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>148</red>
          <green>148</green>
          <blue>148</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/menu/2.png</normaloff>:/menu/2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="lvl_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>148</red>
          <green>148</green>
          <blue>148</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/menu/3.png</normaloff>:/menu/3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <zorder>menu</zorder>
   <zorder>heart_1</zorder>
   <zorder>heart_2</zorder>
   <zorder>heart_3</zorder>
   <zorder>ans_1</zorder>
   <zorder>ans_2</zorder>
   <zorder>ans_3</zorder>
   <zorder>ans_4</zorder>
   <zorder>butt_1</zorder>
   <zorder>butt_2</zorder>
   <zorder>butt_3</zorder>
   <zorder>butt_4</zorder>
   <zorder>tail</zorder>
   <zorder>hero</zorder>
   <zorder>quote</zorder>
   <zorder>level</zorder>
   <zorder>quizz</zorder>
   <zorder>head</zorder>
   <zorder>ans</zorder>
   <zorder>med</zorder>
   <zorder>logo</zorder>
   <zorder>go_away</zorder>
   <zorder>new_game_but</zorder>
   <zorder>lvl_0</zorder>
   <zorder>lvl_1</zorder>
   <zorder>lvl_2</zorder>
   <zorder>lvl_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Игра</string>
    </property>
    <addaction name="new_game"/>
    <addaction name="see_lvl"/>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="new_game">
   <property name="text">
    <string>Новая игра</string>
   </property>
  </action>
  <action name="see_lvl">
   <property name="text">
    <string>Уровни</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
